### PR TITLE
feat(FormById): Implemented retrieving a single form with design data.

### PR DESCRIPTION
**Pull request by nbeijerbacht**
#44

### CODE REVIEW

File: InfoTeamsExperiment/Properties/launchSettings.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
       "commandName": "Project",
       "dotnetRunMessages": "true",
       "launchBrowser": true,
-      "launchUrl": "https://teams.microsoft.com/l/app/d7e4862a-3699-4bb7-8a31-53e780e18a5f?installAppPackage=true&webjoin=true&appTenantId=a4902f43-7b00-4b65-9907-7b084db98898&login_hint=mihail@vfjcd.onmicrosoft.com",
+      "launchUrl": "https://teams.microsoft.com/l/app/fe3469ad-b700-4b9c-ac93-64b0e8e7740f?installAppPackage=true&webjoin=true&appTenantId=a4902f43-7b00-4b65-9907-7b084db98898&login_hint=n.beijerbacht@vfjcd.onmicrosoft.com",
```
Comment (BondarencoM) [inline InfoTeamsExperiment/Properties/launchSettings.json:7]: We should probably need to get rid of some of this stuff, definitely the email.

**Review by BondarencoM — APPROVED** [review]

File: FormService/Models/HeaderRenderer.cs [hunk 2]
```diff
@@ -0,0 +1,25 @@
+﻿using AdaptiveCards;
```
Comment (BondarencoM) [inline FormService/Models/HeaderRenderer.cs:1]: Al these renderers should be in "Logic" as well, or at least not here. They are clearly not models

File: FormService/Models/TextFieldRenderer.cs [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+{
+    public bool CanHandle(Element e) => e.element_type == "field" && e.field.type == "text";
+
+    public IEnumerable<AdaptiveElement> RenderElements(Element e)
+    {
+        yield return new AdaptiveTextBlock
+        {
+            Text = e.text
+        };
+        yield return new AdaptiveTextInput();
```
Comment (BondarencoM) [inline FormService/Models/TextFieldRenderer.cs:16]: We must add the ID here or the render will fail.

**Review by BondarencoM — CHANGES_REQUESTED** [review]

File: InfoTeamsExperiment/Properties/launchSettings.json (hunk 1 above)
Comment (nbeijerbacht) [reply InfoTeamsExperiment/Properties/launchSettings.json:7]: We should either find a solution or we should ignore this file all together.

**Review by BondarencoM — APPROVED** [review]

**Pull request merged**